src/Raven.Client/Documents/Commands/Batches/BatchCommand.cs

```diff
@@ -124,10 +124,7 @@ private void AppendOptions(StringBuilder sb)
             if (_options.WaitForIndexes)
             {
                 sb.Append("&waitForIndexesTimeout=").Append(_options.WaitForIndexesTimeout);
-                if (_options.ThrowOnTimeoutInWaitForIndexes)
-                {
-                    sb.Append("&waitForIndexThrow=true");
-                }
+                sb.Append("&waitForIndexThrow=").Append(_options.ThrowOnTimeoutInWaitForIndexes ? "true" : "false");
```

Comment:
Actually, just calling `ToString()` here is good enough, that is the internal impl detail anyway